onyo/commands/cat.py

```diff
@@ -13,7 +13,7 @@
 log: logging.Logger = logging.getLogger('onyo')
 
 
-def sanitize_paths(paths: list[str], opdir: str) -> list[Path]:
+def sanitize_paths(repo: Repo, paths: list[str]) -> list[Path]:
```

Comment:
If I didn't miss something, `repo` isn't used anymore
```suggestion
def sanitize_paths(paths: list[str]) -> list[Path]:
```

Reply:
I suppose, this needs double-checking across commands - remove unused parameter and adjust calls accordingly.